Packs/ProofpointCasb/ParsingRules/ProofpointCasb/ProofpointCasb.xif

```diff
@@ -0,0 +1,3 @@
+[INGEST:vendor="proofpoint", product="casb", target_dataset="proofpoint_casb_raw", no_hit=drop]
+config case_sensitive=false 
```

Comment:
Need to add a filter below this line, making sure you are only parsing the timestamp you support.